.nvim/nvim-dap.lua

```diff
@@ -1,15 +1,5 @@
 local dap = require('dap')
 
-dap.adapters.cilium_delve = {
-  type = 'server',
-  port = '${port}',
-  executable = {
-    command = 'dlv',
-    args = {'dap', '-l', '127.0.0.1:${port}'},
-    -- add this if on windows, otherwise server won't open successfully
-    -- detached = false
-  }
-}
```

Comment:
It looks like this patch has a bunch of changes merged in that are unrelated to the change you're trying to make (probably you amended the commits to combine a merge commit plus your intended changes at some point). 

If you check out https://github.com/cilium/cilium/commit/73333666fa4e802f6eb51deedcceab9ef6782de9 and fix that one up for any feedback plus do a git rebase, these changes should go away. You can then force push your new copy of the branch over the top of the version in your repository that this PR is being opened from.